src/conf_mode/service_monitoring_telegraf.py

```diff
@@ -159,6 +162,11 @@ def verify(monitoring):
         if 'url' not in monitoring['splunk']:
             raise ConfigError(f'Monitoring splunk "url" is mandatory!')
```

Comment:
`url` should be uppercase

Reply:
The `url` is the missed option of the configuration, not the URL
Do we need to change it to the URL here?
```
vyos@r4# set service monitoring telegraf splunk 
Possible completions:
 > authentication       HTTP basic authentication parameters
   url                  Remote URL

```

Reply:
No, I only mean the URL portion of the error message should be uppercase:

`raise ConfigError('Monitoring splunk URL is mandatory!')`

Reply:
Done!